src/auto-evo/RunResults.cs

```diff
@@ -793,6 +797,24 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                     builder.Append('\n');
             }
 
+            highlightSpecies ??= new HashSet<Species>();
+            if (highlightSpecies.Contains(null))
+                throw new ArgumentNullException("Highlight species entry must not be null");
```

Comment:
`ArgumentNullException` should be reserved to cases where an argument it itself is null, not when an enumerable argument contains null argument. Besides that, it is the reason why Jetbrains (auto-checks) reject the PR, since it expects `ArgumentNullException` to refer to the null argument.

Reply:
Thanks for the info, I already wondered what was wrong with my code